static/src/javascripts/projects/common/utils/closest.js

```diff
@@ -0,0 +1,18 @@
+define(function () {
+    /**
+     * Adds a wrapper around Element.closest()
+     */
+    var matches = 'matches' in Element.prototype ? 'matches' : 'msMatchesSelector';
```

Comment:
Sorry I was late to the party on this but yes I think it's worth adding the `webkit` prefix as the uptake of new versions of Android is way slower than iOS and as such we can expect a fair amount of traffic across versions of Android 4.